docs/data-sources/segments.md

```diff
@@ -0,0 +1,39 @@
+# configcat_segments Resource
+
+Use this data source to access information about existing **Segments**.
```

Comment:
```suggestion
Use this data source to access information about existing **Segments**. Read more about segmentation and segments in ConfigCat [here].
```

Reply:
A többihez hasonlóan "What is a Segment.." lett a leírás.